torch/testing/_internal/custom_op_db.py

```diff
@@ -435,3 +435,54 @@ def sample_inputs_numpy_nms(opinfo, device, dtype, requires_grad, **kwargs):
         supports_out=False,
     ),
 ]
+
+
+# ==============================================================
+# some mechanical test cases
+# ==============================================================
+
+lib = torch.library.Library("_torch_testing", "FRAGMENT")  # noqa: TOR901
```

Comment:
Wait this would always get registered if anyone end up importing this by mistake with torch. ?

Reply:
Yes. All the custom ops in this file get registered if they get imported. It's okay because they're in a namespace that users shouldn't use.